explorer/src/test/java/com/scalar/client/tool/explorer/ExplorerTest.java

```diff
@@ -44,196 +41,195 @@
 
   @Rule public ExpectedException thrown = ExpectedException.none();
 
   @Before
   public void setUp() {
     MockitoAnnotations.initMocks(this);
 
+    clientExceptionCertificateNotFound =
+            new ClientException("Certificate not found", StatusCode.CERTIFICATE_NOT_FOUND);
+    clientExceptionContractNotFound =
+            new ClientException("contract not found", StatusCode.CONTRACT_NOT_FOUND);
     explorer = new Explorer(clientService, clientConfig);
-    when(clientConfig.getCertHolderId()).thenReturn(HOLDER_ID);
 
-    when(clientService.registerCertificate()).thenReturn(ledgerServiceResponseOK);
-    when(clientService.registerContract(
-            anyString(), anyString(), anyString(), eq(Optional.empty())))
-        .thenReturn(ledgerServiceResponseOK);
-
-    when(ledgerServiceResponseOK.getStatus()).thenReturn(StatusCode.OK.get());
-    when(ledgerServiceResponseCertificateNotFound.getStatus())
-        .thenReturn(StatusCode.CERTIFICATE_NOT_FOUND.get());
-    when(ledgerServiceResponseCertificateNotFound.getMessage()).thenReturn("certificate not found");
-
-    when(contractExecutionResponseOK.getStatus()).thenReturn(StatusCode.OK.get());
-    when(contractExecutionResponseCertificateNotFound.getStatus())
-        .thenReturn(StatusCode.CERTIFICATE_NOT_FOUND.get());
-    when(contractExecutionResponseCertificateNotFound.getMessage())
-        .thenReturn("certificate not found");
-    when(contractExecutionResponseContractNotFound.getStatus())
-        .thenReturn(StatusCode.CONTRACT_NOT_FOUND.get());
-    when(contractExecutionResponseContractNotFound.getMessage()).thenReturn("contract not found");
+    when(clientConfig.getCertHolderId()).thenReturn(HOLDER_ID);
   }
 
   @Test
-  public void get_CertificateAndContractNotRegistered_ShouldSucceed() {
+  public void get_ProperArgumentGiven_ShouldSucceed() {
     // Arrange
-    when(contractExecutionResponseOK.getResult()).thenReturn("{ \"id\": \"bar\" }");
+    JsonObject expected = Json.createObjectBuilder().add("status", StatusCode.OK.get()).build();
+    when(contractExecutionResultOK.getResult()).thenReturn(Optional.of(expected));
     when(clientService.executeContract(eq(HOLDER_ID + "GET"), any(JsonObject.class)))
-        .thenReturn(
-            contractExecutionResponseCertificateNotFound,
-            contractExecutionResponseContractNotFound,
-            contractExecutionResponseOK);
+        .thenReturn(contractExecutionResultOK);
 
     // Act
-    JsonObject result = explorer.get(ASSET_ID);
+    JsonObject actual = explorer.get(ASSET_ID);
 
     // Assert
-    verify(clientService, Mockito.times(3))
-        .executeContract(eq(HOLDER_ID + "GET"), any(JsonObject.class));
-    assertThat(result.getString("id")).isEqualTo("bar");
+    verify(clientService).executeContract(eq(HOLDER_ID + "GET"), any(JsonObject.class));
+    assertThat(actual).isEqualTo(expected);
   }
 
   @Test
   public void get_FailedToRegisterCertificate_ShouldThrowExplorerException() {
```

Comment:
Please update method names.

Reply:
Thank you for review change.
Method names updated in 135ba8e